### PR TITLE
Fix subheader

### PR DESCRIPTION
**Pull request by LeoHabs**
Fixed the white space wrapping

### CODE REVIEW

File: src/redux/actions/BudgetActions.ts [hunk 1]
```diff
@@ -25,41 +35,47 @@ export const ListBudgetAction = () => async (dispatch: Dispatch) => {
...
+          item?.budgetProducts?.reduce(
+            (acc: any, entry: { quantity: any }) =>
+              (acc += entry?.quantity ?? 0),
+            0
+          ) ?? 0;
         return { ...item, budgetProducts: quantity };
-        });
-        console.log(newData);
+      });
+      console.log(newData);
```
Comment (Gabriel-Sous-a) [inline src/redux/actions/BudgetActions.ts:47]: sneaky console.log

Comment (Gabriel-Sous-a) [inline src/redux/actions/BudgetActions.ts]:
```suggestion
      
```

sneaky

**Review by Adan0k — CHANGES_REQUESTED** [review]
tu ta fazendo pr pra main

**Review by Adan0k — APPROVED** [review]
boa

**Review by Gabriel-Sous-a — APPROVED** [review]

**Pull request merged**